Expensify/App
# Improved UX when pasting images and there is an error

### Details
1. Added a global Growl Component which can be called from any component.
2. Show growl on failed parsing of image & allow pasting of text.

> NOTE: Pasting of text will include the image name as well as this is how browser parse the copied content. 

### Fixed Issues

Fixes #2530

### Tests / QA Steps
1. Open E.cash on web.
2. Copy any content which include any text & image that can't be parsed in E.cash.
3. Now paste that on composer. Observe that error growl should be shown for failed image parsing & text should be pasted on the composer.
4. Copy any image that can parsed by E.cash. Pasting it on composer should trigger image Modal.
5. Copy any text & pasting it on composer should work as normal.

A good example of **any image & text that can't be parsed** can be seen on the video linked for web. It can be found here https://github.com/Expensify/Expensify.cash/issues/2530.

### Tested On

- [x] Mobile Web (Not affected)

### Screenshots


#### Web


https://user-images.githubusercontent.com/24370807/119171601-6b153980-ba82-11eb-8aae-26081942272a.mp4


#### Mobile Web


#### Desktop

 Can't find any such image which fails to parse on desktop app. :man_shrugging: 
#### iOS


#### Android


https://user-images.githubusercontent.com/24370807/119887350-a798d780-bf51-11eb-9dcb-a518bfd82f3e.mp4
